### PR TITLE
Excluded 'TaskStatus.STARTING' from task progress check

### PR DESCRIPTION
**Pull request by yarongilor**
this workaround fix should resolve: 
sctool task progress fails #856

### CODE REVIEW

File: sdcm/mgmt.py [hunk 1]
```diff
@@ -173,7 +173,8 @@ def is_status_in_list(self, list_status, check_task_progress=False):
         :return:
         """
         status = self.status
-        if check_task_progress and status != TaskStatus.NEW:  # check progress for all statuses except 'NEW'
+        if check_task_progress and status not in [TaskStatus.NEW,
+                                                  TaskStatus.STARTING]:  # check progress for all statuses except 'NEW' / 'STARTING'
             progress = self.progress
```
Comment (bentsi) [inline sdcm/mgmt.py]: where is a progress variable used?

Comment (roydahan) [reply sdcm/mgmt.py]: it's not used.

**Comment by roydahan** [comment]
@yarongilor is it tested and works?

File: sdcm/mgmt.py (hunk 1 above)
Comment (bentsi) [reply sdcm/mgmt.py]: @yarongilor so what is the point of this if? to printout/run `task progress`?

Comment (yarongilor) [reply sdcm/mgmt.py]: the reasons are:
1) check that progress command works on varios task statuses ( that was how last manager bug found).
2) print the progress to log in cases needed for failures/performance analysis.

Comment (bentsi) [reply sdcm/mgmt.py]: It is a bit confusing and not understandable from the code. I would add a comment with a reason why this is done.

**Comment by yarongilor** [comment]
The fix tested and pass ok in centos sanity, build 43 :
http://jenkins.cloudius-systems.com:8080/job/mermaid-1.3/job/mermaid-1.3-centos-sanity-test/

**Comment by roydahan** [comment]
@bentsi please backport to manager-v1.3.1 branch.

File: sdcm/mgmt.py (hunk 1 above)
Comment (bentsi) [reply sdcm/mgmt.py]: @yarongilor please add the reasons that you mentioned as the comments to the code!!!

Comment (yarongilor) [reply sdcm/mgmt.py]: done. comment added.

**Comment by bentsi** [comment]
Thanks @yarongilor

**Pull request merged**